lambda_functions/rds_init/lambda-handler.py

```diff
@@ -0,0 +1,82 @@
+import json
+import logging
+import os
+import sys
+
+import psycopg2
+from aws_lambda_powertools.utilities import parameters
+from aws_lambda_powertools.utilities.typing import LambdaContext
+
+# ----- Production BDE -----
+bde_host = os.environ["BDE_HOST"]
+bde_analytics_user_secret = os.environ["BDE_ANALYTICS_USER_SECRET"]
+bde_analytics_user_secret = json.loads(bde_analytics_user_secret)
+
+bde_analytics_user_name = bde_analytics_user_secret["username"]
+bde_analytics_user_pw = bde_analytics_user_secret["password"]
+
+
+# ----- FDW Analytics -----
+rds_fdw_host = os.environ["RDS_FDW_HOST"]
+rds_fdw_db = os.environ["RDS_FDW_DB"]
+
+rds_fdw_root_cred = parameters.get_secret(os.environ["RDS_FDW_ROOT"])
+
+rds_fdw_root_cred = json.loads(rds_fdw_root_cred)
+rds_fdw_root_user = rds_fdw_root_cred["username"]
+rds_fdw_root_pw = rds_fdw_root_cred["password"]
+
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def connect_prod_bde_db() -> psycopg2.connection:  # pylint: disable=no-member
+    try:
+        conn = psycopg2.connect(
+            host=rds_fdw_host,
+            database=rds_fdw_db,
+            user=rds_fdw_root_user,
+            password=rds_fdw_root_pw,
+        )
+        logger.info("SUCCESS: Connection to POSTGRES RDS instance succeeded")
+        return conn
+
+    except psycopg2.Error as e:
+        logger.error("%s error: Unable to connect to Postgresql RDS: %s due to: %s", type(e), rds_fdw_host, e)
+        sys.exit()
```

Comment:
Idiomatically you probably want to `raise` instead of `sys.exit()` - It enables callers to handle the error, and gives a non-zero exit code.